src/lib/types.ts

```diff
@@ -581,6 +581,10 @@ export interface WalletData {
   new_to_crypto?: boolean
 }
 
+export interface Wallet extends WalletData {
+  supportedLanguages: string[]
+}
+
```

Comment:
Could we use a type here? (it's inside `types.ts`)

```suggestion
export type Wallet = WalletData & {
  supportedLanguages: string[]
}

```

Reply:
Sure. I think we used more intersections than extensions, right? I did that because `WalletData` is already an interface.